content/v2.0/account-management/_index.md

```diff
@@ -4,6 +4,9 @@ description: >
   View and manage information related to your InfluxDB Cloud 2.0 account such as
   pricing plans, data usage, account cancelation, etc.
 weight: 4
+cloud_all: true
```

Comment:
I removed the `cloud_all` frontmatter from all the templates, so this won't do anything. Replace it with `products: [cloud]`.